yt_dlp/extractor/nbc.py

```diff
@@ -578,6 +589,7 @@ class NBCOlympicsStreamIE(AdobePassIE):
             'params': {
                 'skip_download': 'm3u8',
             },
+            'skip': 'redirects to nbcolympics.com. Possibly geo-restricted?',
```

Comment:
this extractor doesn't use ThePlatform

Reply:
I realize, I just figured I'd fix it while I'm at it. But otherwise, see comment above.